LoopUI/Extensions/Color.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Color.swift
+//  LoopUI
+//
+//  Created by Nathaniel Hamming on 2020-07-28.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+// Color version of the UIColor context colors
+extension Color {
+    public static let agingColor = Color(UIColor.agingColor)
```

Comment:
I'm wondering if these should use the Color constructor that does the asset catalog lookup directly, rather than constructing an intermediate UIColor. i.e. `Color("warning")`.  I'm not sure that using the UIColor constructor will allow the Color struct to react to color scheme changes in the same way.

Reply:
Updated and there are only 3 colours (grid, cellBackground, and unknown) that still pull from `UIColor`.